### PR TITLE
Log conda version and docker image hash when creating a container.

### LINKED ISSUE #8375: Logging: log conda version and docker image hash
While testing #8328, I found a couple of things that could be logged to improve observability:

- [x] Log conda version, if used
- [ ] Log Docker image hash, apart from `stable`, `latest`, or `testing`

### PR DESCRIPTION
**Pull request by dogukanteber**
This PR hopefully resolves #8375 . I have written a small method to get the conda version but I also saw venv_path in the Conda class which is located in python_environment.py. I could not decide if it returns the conda version. If it is more appropriate, I can remove the method I wrote and change it with the existing one.

### CODE REVIEW

**Review by humitos — CHANGES_REQUESTED** [review]
Thanks for your PR! I'm requesting changes because we need to run the command inside the Docker container. We can't use `subprocess.run` for this unfortunately.

File: readthedocs/doc_builder/environments.py [hunk 1]
```diff
@@ -977,6 +977,20 @@ def container_id(self):
...
+    @property
+    def conda_version(self):
+        """Return conda version if exists."""
+        try:
+            version = subprocess.run(
+                ["conda", "--version"],
+                capture_output=True,
+                text=True,
+                check=True,
+            )
```
[the remark this answers is not in the document]
Comment (dogukanteber) [reply readthedocs/doc_builder/environments.py:989]: Thank you for your comment @humitos. I am dealing with my finals now. Would that be okay if a take a look at it in the next week?

[the remark this answers is not in the document]
Comment (humitos) [reply readthedocs/doc_builder/environments.py:989]: Yes! 👍🏼

[the remark this answers is not in the document]
Comment (dogukanteber) [reply readthedocs/doc_builder/environments.py]: I have made some changes @humitos.

**Comment by astrojuanlu** [comment]
Hi @dogukanteber ! Some tests are still failing: `E       AttributeError: 'NoneType' object has no attribute 'container_id'` (see the red Circle CI check. Have you already finished your finals? :)

**Comment by dogukanteber** [comment]
Hello @astrojuanlu. Yes, I have finished my finals. I have not realized some tests failed. I did some changes but it did not work. Right now, I am stuck. I cannot find a way out. Do you have any recommendations on how should I proceed?

**Comment by ericholscher** [comment]
Thanks for the PR. This can now be accomplished with `build.jobs` config option, so I'm going to close this.

https://docs.readthedocs.io/en/latest/config-file/v2.html#build-jobs

**Pull request closed without merging**